docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -0,0 +1,180 @@
+|`id` |String |The item's unique identifier. a|Yes, when the item's `item_id` field is not used.
+|`item_id` |String a|The `item_id` of the item you are updating. a|Yes, when the item's `id` field is not used.
+|`meta` |Object |Placeholder for metadata about the exception item. |No
+|`name` |String |The {endpoint-artifact-name-single} name. |Yes
+|`namespace_type` a|String a|Must be set to `agnostic`. |Yes
+|`tags` |String[] |An array of strings containing words and phrases to help categorize
+exception items. Tags can also be used to define the {endpoint-artifact-name-single} as either globally applicable to all policies or assigned to one or more policies (per policy). See xref:exception-item-scope-assignment-tags[] for more details. |No
+|`type` |String a|Must be `simple`. |Yes
+|`_version` |String a|The version id, normally returned by the API when the item was retrieved for update. |Yes
+
+|==============================================
+// end::update-exception-item-request-body[]
+
+
+
+// tag::api-id-or-itemid-query-params[]
+The URL query must include one of the following:
+
+* `id` - `/api/exception_lists/items?id=<id>`
+* `item_id` - `/api/exception_lists/items?item_id=<item_id>`
+// end::api-id-or-itemid-query-params[]
+
+
+// tag::find-exception-items-query-params[]
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`list_id` |String a|Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
+|`namespace_type` |String a|Must be set to `agnostic`. |Yes
+|`page` |Integer |The page number to return. |No
+|`per_page` |Integer |The number of items to return per page. |No
+|`sort_field` |String |Determines which field is used to sort the results. |No
+|`sort_order` |String a|Determines the sort order, which can be `desc` or `asc`. |No
+|`filter` |String |A {kibana-ref}/kuery-query.html[{kib} Query Language (KQL)] string to filter the results down. |No
+
+|==============================================
+// end::find-exception-items-query-params[]
+
+
+
+// tag::endpoint-exceptions-value-types-info-section[]
+==== Value types
+
+The following describes the types that can be defined when using the xref:create-item-api[create] or xref:update-item-api[update] {endpoint-artifact-name} APIs:
+
+[#exception-item-comment-schema]
+===== `comment` object schema
+
+Comments are JSON objects containing the following structure:
+
+[source,json]
+---------
+{
+    "comment": "some comment here"
+}
+---------
+
+When used with the update API, existing comments can be updated by using their associated `id`, while any comment without the `id` attribute will be added as a new comment:
+
+[source,json]
+---------
+{
+    "comment": "some comment here - updated",
+    "id": "1078cf59-5893-4143-acf7-40a40af16bee"
+}
+---------
```

Comment:
Should we make this conditional so it applies only to Event Filters? We can't add/update comments to other artifacts via UI at least.

Reply:
Good point. I think the API allows it (I have not tried it), so maybe we just leave it. I have a feeling we'll introduce comments for all artifacts eventually

Reply:
Yeah the API allows adding/updating comments.

Reply:
I say we leave it. Else, we'll need to adjust the docs to indicate things like "set it to empty array. Not currently used" when really its not true.

Reply:
That is a fair point. 👍